lib/binding/kvdb_interface.c

```diff
@@ -411,7 +409,11 @@ hse_init(const char *const config, const size_t paramc, const char *const *const
     }
 
     /* First log message w/ HSE version - after deserializing global params */
-    log_info("version %s, program %s", HSE_VERSION_STRING, hse_progname);
+#ifdef _GNU_SOURCE
+    log_info("version %s, program %s", HSE_VERSION_STRING, program_invocation_short_name);
```

Comment:
One could argue how useful it is to log the program name when syslog already takes care of this if you don't call openlog() with a custom identifier.

Just a thought.

Reply:
The original intent was to print the full path, so maybe change this to "program_invocation_name".

Reply:
Ok, I will do that.